lib/View/download_screen.dart

```diff
@@ -0,0 +1,50 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:gocast_mobile/View/utils/custom_bottom_nav_bar.dart';
+import 'package:gocast_mobile/View/utils/video_card_view.dart';
+
+class DownloadsScreen extends ConsumerWidget {
+  const DownloadsScreen({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text('Downloads'),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.search),
+            onPressed: () {
+              // Implement search functionality
+            },
+          ),
+          IconButton(
+            icon: const Icon(Icons.filter_list),
+            onPressed: () {
+              // Implement more options functionality
+            },
+          ),
+        ],
+      ),
+      body: ListView(
```

Comment:
Refactor to use component (see comment in `bookmarks_screen).

Reply:
done